### PR TITLE
Add Tor documentation

### PR DESCRIPTION
**Pull request by Stadicus**

### CODE REVIEW

File: docs/support/tor.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Revealing your location with the information that you are using Bitcoin is not a good idea. This is why the BitBox Base uses [The Onion Router](https://www.torproject.org/) (Tor) to communicate to other Bitcoin peers privately and without revealing your real IP address.
+
+By creating "hidden services", Tor can also be used to securely access your BitBox Base from outside your local network without having to configure any networking devices, such as port-forwarding on your router. 
+
+Please not that only Bitcoin-related communication is routed over Tor.
+
+### Installation
+
+Tor is installed using the official repository at `https://deb.torproject.org/torproject.org`, over a TLS connection secured by the official Tor signing key.
```
Comment (hkjn) [inline docs/support/tor.md]: I think it's not quite accurate to say that the `official Tor signing key` is what secures the `TLS connection` (transport-layer security is separate from code signing).. maybe:

> Tor is installed using the official repository at `https://deb.torproject.org/torproject.org`, and the official PGP signing key for the Tor project is used to verify the integrity of the package.

File: docs/support/tor.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### Configuration
+
+The Tor configuration is application-specific:
+
+* **Bitcoin Core**
+  * Currently, `bitcoind` is not configured to listen to other Tor nodes.
+  * To connect to other Bitcoin nodes, all communication is routed over the Tor SOCKS proxy. This configuration is set in `/etc/bitcoin/bitcoin.conf` with the option `proxy=127.0.0.1:9050`. Peers will see a regular IP address, but one that belongs to the Tor network and not your own.
+  * For initial peer discovery on the Tor network, well-known `.onion` nodes are provided to get additional addresses from:
```
Comment (hkjn) [inline docs/support/tor.md]: Do we have any info on where we found this list of "well-known" `.onion` nodes?

Comment (Stadicus) [reply docs/support/tor.md]: Unfortunately not, these are used everywhere in scripts on the internet, maybe @jonasschnelli knows them?

File: docs/support/tor.md (hunk 1 above)
Comment (Stadicus) [reply docs/support/tor.md]: Changed to 
> Tor is installed using signed Debian packages from the official repository at `https://deb.torproject.org/torproject.org`.

File: docs/support/tor.md (hunk 2 above)
Comment (Stadicus) [reply docs/support/tor.md]: I'll open an issue for that, but I think it's safe to merge this as-is.

Comment (Stadicus) [reply docs/support/tor.md]: https://github.com/shiftdevices/bitbox-base-internal/issues/180

File: docs/support/tor.md (hunk 1 above)
Comment (hkjn) [reply docs/support/tor.md]: Seems to have been done in https://github.com/digitalbitbox/bitbox-base/pull/66. Resolving this thread.

File: docs/support/tor.md (hunk 2 above)
Comment (hkjn) [reply docs/support/tor.md]: Okay, sounds good. Resolving this thread.

**Pull request merged**